CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+ - Fix: Corrected the image urls and base_uri that were broken in the build
```

Comment:
Suggestion: Fixed an issue that caused incorrect URL paths if the project gets built.